client/add.go

```diff
@@ -22,16 +31,33 @@ For more about knox, see https://github.com/pinterest/knox.
 See also: knox create, knox promote
 	`,
 }
+var addTinkKeyset = cmdAdd.Flag.String("key-template", "", "")
```

Comment:
nit: the third string here is a usage string, so you could add information about what this flag is / how it is used here.

Reply:
Thank you for your comment! I revised the code accordingly.